### ISSUE TITLE
Archive 2015 MozFest site

### ISSUE THREAD

**Issue by mmmavis** [MozillaFoundation/mofo-devops#309]
@cadecairos (sorry not sure who else to tag here :grin: )

Currently https://mozillafestival.org/ goes https://2015.mozillafestival.org/ We are going to update the site with 2016 info soon and I think we should probably archive the 2015 site first. Not sure how we want to do this though...

1) Do we want to leave 2015 site as is, create new instance and repo for the 2016 site, have it up on 2016.mozillafestival.org and points mozillafestival.org to it ?

OR

2) crawl all the pages on 2015 site, repurpose the existing [2015 site repo](https://github.com/mozilla/2015.mozillafestival.org) and Heroku instances for 2016 site?

OR

... ?

/cc @edrushka @ScottDowne

**Comment by ScottDowne** [comment]
For what it's worth, last year we did item number 1.

**Comment by simonwex** [comment]
For now, let's leave the 2015 site as-is. File a separate issue here in mofo-devops to archive it and host it on S3/cloudfront. – This is purely a cost-savings measure. At the same time, we should create a branch of the current codebase as `2015.mozillafestival.org`.– We can use this to make any changes we like to the old site. 

We can continue development of the new site in this repo: https://github.com/mozilla/2015.mozillafestival.org but we should rename the repo to simply `mozillafestival.org`. We can then create a new heroku instance for staging and production that will answer to mozillafestival.org and redirect from www.mozillafestival.org.

Make sense?

**Comment by xmatthewx** [comment]
Makes total sense, as long as work on the new site isn't blocked by the need archive. If branches allow for that, great. If not, let's fork it. 

 - [ ] `p1` enable dev of 2016 site at root `mozillafestival.org`
 - [ ] `p2` archive 2015 site

**Comment by mmmavis** [comment]
I'll update this ticket & create another ticket once I get r+ from stakeholders on the [staging site](https://mozillafestival-org-staging.herokuapp.com)

**Comment by mmmavis** [comment]
- `p1` enable dev of 2016 site at root `mozillafestival.org` 
(update: ticket entered https://github.com/MozillaFoundation/mofo-devops/issues/316 )

- `p2` archive 2015 site
(will edit this ticket title & description a bit)

**Comment by mmmavis** [comment]
Created 2 follow-up tickets based on our discussion above. Closing this one now.
